tutorials/advanced/transforms_tutorial.py

```diff
@@ -3,80 +3,94 @@
+|"figure"        |The coordinate system of the       |``fig.transFigure``          |
+|                |`.Figure`; (0, 0) is bottom left   |                             |
+|                |of the figure, and (1, 1) is top   |                             |
+|                |right of the figure.               |                             |
++----------------+-----------------------------------+-----------------------------+
+|"figure-inches" |The coordinate system of the       |``fig.dpi_scale_trans``      |
+|                |`.Figure` in inches; (0, 0) is     |                             |
+|                |bottom left of the figure, and     |                             |
+|                |(width, height) is the top right   |                             |
+|                |of the figure in inches.           |                             |
++----------------+-----------------------------------+-----------------------------+
+|"xaxis",        |Blended coordinate systems; use    |``ax.get_xaxis_transform()``,|
+|"yaxis"         |data coordinates on one of the axis|``ax.get_yaxis_transform()`` |
+|                |and axes coordinates on the other. |                             |
++----------------+-----------------------------------+-----------------------------+
+|"display"       |The native coordinate system of the|``None``, or                 |
+|                |output ; (0, 0) is the bottom left |``IdentityTransform()``      |
+|                |of the window, and (width, height) |                             |
+|                |is top right of the output in      |                             |
+|                |"display units".                   |                             |
+|                |                                   |                             |
+|                |The exact interpertation of the    |                             |
+|                |units depends on the back end. For |                             |
+|                |example it is pixels for Agg and   |                             |
+|                |pts for svg/pdf.                   |                             |
++----------------+-----------------------------------+-----------------------------+
+
+
+
+
+
+The `~matplotlib.transforms.Transform` objects are naive to the source and
+destination coordinate systems, however the objects refereed to in the table
+above are constructed to take inputs in their coordinate system, and transform
+the input to the *display* coordinate system.  That is why the *display*
+coordinate system has ``None`` for the ``Transformation Object`` column -- it
+already is in *display* coordinates.  The naming and destination conventions
+are an aid to keeping track of the available "standard" coordinate systems and
+transforms.
+
+The transformations also know how to invert themselves, to go from *display*
+back to the native coordinate system.  This is particularly useful when
```

Comment:
```suggestion
The transformations also know how to invert themselves, to go from *display*
back to the transform's coordinate system.  This is particularly useful when
```

not sure this is right, but "native" isn't fully clear here...

Reply:
back to the input coordinate system?